apollo-api-impl/src/main/java/com/spotify/apollo/request/TrackingEndpointRunnableFactory.java

```diff
@@ -26,6 +26,7 @@
 /**
  * An {@link EndpointRunnableFactory} that collect statistics
  */
+@Deprecated
```

Comment:
It's still called from [`Handlers#withTracking()`](https://github.com/spotify/apollo/pull/87/files#diff-a9d3e4572f3699295840da558f6d80b6R63) though, so unless we want to break the public api now I guess we need to keep it around?